minecraft/p2p/example_client_test.go

```
@@ -74,30 +71,17 @@ func ExampleClient() {
-		s = conn
-	default:
-		panic(fmt.Sprintf("invalid connection type: %d", connection.Type))
+	target, err := session.ClientTarget()
+	if err != nil {
+		panic(fmt.Sprintf("error preparing client target: %s", err))
 	}
+	s, err := target.DialSignaling(context.TODO(), src, ClientSignalingOptions{})
+	if err != nil {
+		panic(fmt.Sprintf("error dialing client signaling: %s", err))
+	}
+	defer s.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `s.Close` is not checked

(errcheck)